doc/dev/installation.rst

```diff
@@ -8,7 +8,7 @@ pre-built binaries are being distributed via the Python Package Index (PyPI) for
 
 .. code-block:: console
 
-    pip install pennylane-catalyst
+    pip install PennyLane-Catalyst
```

Comment:
does this need to change? by convention most pip commands are lowercase

Reply:
Reverted